lib/rspec/mocks.rb

```diff
@@ -1,7 +1,6 @@
 require 'rspec/support'
 RSpec::Support.require_rspec_support 'caller_filter'
 RSpec::Support.require_rspec_support 'warnings'
-RSpec::Support.require_rspec_support 'ruby_features'
```

Comment:
Honestly I would have left this here.

Reply:
Couldn't quickly find the issue, but I recall a weird case when `rspec-mocks` were required partially.
As an example, [this](https://github.com/rspec/rspec-mocks/blob/461d7f6f7f688f69154e410dcd9c7690120e7dbf/lib/rspec/mocks/argument_list_matcher.rb#L4):
```ruby
require 'rspec/mocks/argument_matchers'
```
Where [`lib/rspec/mocks/argument_list_matcher.rb `](https://github.com/rspec/rspec-mocks/blob/fa45affa48d2dadcd67d15644d403dedd601dfd5/lib/rspec/mocks/argument_matchers.rb#L4) don't require `rspec/mocks`.

I've moved `RSpec::Support.require_rspec_support 'ruby_features'` where `RubyFeatures` are used.